tests/v2_tests/test_create_index.py

```diff
@@ -38,10 +38,13 @@ def test_simple_index_creation(self):
         expected_settings = {
```

Comment:
Can you have at least one test on the error message?

Reply:
done